vtex-apps/product-summary
# Changing listName to be sent to pdp via params

#### What problem is this solving?

Product Summary now sends the `listName` to the PDP via runtime params. This PR adds this behavior to legacy Product Summaries and also re-enables de default `true` behavior of the `trackListName` prop. `listName` is now invisible to the user and is more appropriate for this use case.

#### How to test it?

Click any shelf or search result. It should take you to the product page. Type `dataLayer` into your console, and check for the last `productDetail` event. It should have the name of the list you clicked somewhere there. The default listName for shelves is `List of products`.

[Workspace](https://icarogtm--storecomponents.myvtex.com/)

#### Describe alternatives you've considered, if any.

I tried putting it on the URL, it broke some marketing campaigns and clients complained about it.

#### Related to / Depends on

Depends on [https://github.com/vtex-apps/product-summary-context/pull/19](https://github.com/vtex-apps/product-summary-context/pull/19)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/s4kpsTSwoiFCU/giphy.gif)
